src/index.js

```diff
@@ -134,6 +134,10 @@ const printers = {
         {
           kinds: ["parameter"],
           children: { listNodes: [], nodes: ["value"] }
+        },
+        {
+          kinds: ["retif"],
+          children: { listNodes: [], nodes: ["trueExpr", "falseExpr"] }
```

Comment:
should probably add [test](https://github.com/glayzzle/php-parser/blob/master/docs/AST.md#retif) as well

Reply:
Good catch! done.